cluster/swarm/cluster.go

```diff
@@ -221,8 +221,17 @@ func (c *Cluster) createContainer(config *cluster.ContainerConfig, name string,
 
 	c.scheduler.Unlock()
 
-	log.WithFields(log.Fields{"NodeName": n.Name, "NodeID": n.ID}).Debugf("Scheduling container %s to ", name)
 	container, err := engine.CreateContainer(config, name, true, authConfig)
+	containerFlag := name
```

Comment:
nit: These few lines are for `err==nil`. It'd be better to put them in the `else` below.